MapboxCoreNavigation/RouteController.swift

```diff
@@ -287,7 +288,7 @@ open class RouteController: NSObject {
         let step = stepProgress.step
         
         //Increment the progress model
-        let polyline = Polyline(step.coordinates!)
+        let polyline = Polyline(step.shape!.coordinates)
```

Comment:
Avoid copying the Polyline:

```suggestion
        let polyline = step.shape!
```